pom.xml

```diff
@@ -94,6 +94,12 @@
             <artifactId>elasticsearch</artifactId>
```

Comment:
Do we still need the dependency on `org.elasticsearch:elasticsearch`?
The High Level REST client brings that as a transitive dependency.

Reply:
removed